splinepy/io/gismo.py

```diff
@@ -17,6 +17,7 @@ def _spline_to_ET(
     index_offset,
     fields_only=False,
     as_base64=False,
+    field_mask=True,
```

Comment:
```suggestion
    field_mask=None,
```
Good point, thanks @j042